projects.yaml

```diff
@@ -190,3 +190,27 @@ projects:
         - *module_if
         - *module_hic_lpc11u35
         - records/board/archlink.yaml
+    lpc11u35_6LoWPAN_BorderRouterUSB_if:
+        - *module_if
+        - *module_hic_lpc11u35
+        - records/board/6LoWPAN_BorderRouterUSB.yaml
+    lpc11u35_6LoWPAN_BorderRouterHAT_if:
+        - *module_if
+        - *module_hic_lpc11u35
+        - records/board/6LoWPAN_BorderRouterHAT.yaml
+    lpc11u35_6LoWPAN_BorderRouterPOE_if:
+        - *module_if
+        - *module_hic_lpc11u35
+        - records/board/6LoWPAN_BorderRouterPOE.yaml
+ lpc11u35_swdap_k64f_if:
+        - *module_if
+        - *module_hic_lpc11u35
+        - records/board/swdap_k64f.yaml
+    lpc11u35_swdap_lpc1768_if:
```

Comment:
The SWDAP is intentionally created without a target since it is meant to be used as a target independent debugger. This probably needs some further discussion.

CC: @chris-styles, @sg-